### PR TITLE
Add comment on last milestone merged

### PR DESCRIPTION
**Pull request by 0xCaso**
Now `badge.yml` is `badge_or_last_milestone.yml`

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]

File: .github/workflows/badge_or_last_milestone.yml [hunk 1]
```diff
@@ -81,3 +87,26 @@ jobs:
...
+    steps:
+      - name: Last milestone comment
+        uses: thollander/actions-comment-pull-request@1.0.2
+        with:
+          message: > 
+            We noticed that this is the last milestone of your project, so what are the potential next steps?<br/>
+            
+            While you can always apply for a follow-up grant, the grants program might not be the best tool for your future growth. 
+            Our main goal is to support research as well as early-stage technical projects. Below are a couple of other options in our ecosystem 
+            that might be better suited as the next step: <br/>
```
Comment (keeganquigley) [inline .github/workflows/badge_or_last_milestone.yml:104]:
```suggestion
            that might be better suited as the next step: <br/> 
```
```suggestion
            that might be better suited as the next step, depending on your goals: <br/> 
```

File: .github/workflows/badge_or_last_milestone.yml [hunk 2]
```diff
@@ -81,3 +87,26 @@ jobs:
...
+            While you can always apply for a follow-up grant, the grants program might not be the best tool for your future growth. 
+            Our main goal is to support research as well as early-stage technical projects. Below are a couple of other options in our ecosystem 
+            that might be better suited as the next step: <br/> 
+
+            Project with a Business Case/Token: [Substrate Builders Program](https://substrate.io/ecosystem/substrate-builders-program/) 
+            or [VC Funding](https://parachains.info/investors) <br/> 
+
+            Common Good Projects: [Treasury Funding](https://github.com/w3f/Grants-Program#treasury) <br/> 
+
+            Let us know if you have any questions regarding the above. We are more than happy to help you with it.
```
Comment (keeganquigley) [inline .github/workflows/badge_or_last_milestone.yml:111]:
```suggestion
            Let us know if you have any questions regarding the above. We are more than happy to help you determine the best course of action. 
```

Comment (keeganquigley) [reply .github/workflows/badge_or_last_milestone.yml:111]: Maybe this is going too far but I figure we can at least help them determine some of the differences between funding sources.

Comment (Noc2) [reply .github/workflows/badge_or_last_milestone.yml:111]: No. I think this makes sense for teams that completed a grant. Thanks!

**Pull request merged**